ext/sockets/tests/socket_cmsg_credentials_fbsd.phpt

```diff
@@ -0,0 +1,92 @@
+--TEST--
+recvmsg(): receive SCM_CREDS messages
+--EXTENSIONS--
+sockets
+--SKIPIF--
+<?php
+
+if (!str_contains(PHP_OS, 'FreeBSD')) {
+    die('skipped supported only on FreeBSD');
```

Comment:
While that would indeed skip the test, no reason would be reported:
https://github.com/php/php-src/blob/9c6b843ff0f4b7af926a69ef89d7f9ec1db31866/run-tests.php#L2194-L2199